docs/collecting-data/collecting-from-own-applications/webview-tracker/index.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "WebView Tracker"
+date: "2020-02-26"
+sidebar_position: 135
+---
+
+```mdx-code-block
+import Badges from '@site/src/components/Badges';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Badges badgeType="Early Release"></Badges>
+```
+
+The [Snowplow WebView Tracker](https://github.com/snowplow-incubator/snowplow-webview-tracker) allows you to track Snowplow events from Web views in **mobile hybrid apps**.
+
+Hybrid apps are mobile apps that in addition to a native interface, provide part of the UI through an embedded Web view.
+Snowplow events are tracked from both the native code (e.g. written in Swift or Kotlin) as well as the Web view (in JavaScript).
+Our goal is to have both events tracked from the native code as well as the Web view, share the same session and appear as tracked with the same tracker.
+
+The WebView tracker should be integrated in Web apps used in Web views within native mobile apps.
+The tracker provides APIs to track Snowplow events.
+It forwards the events to the native app code to be tracked by the Snowplow mobile trackers ([iOS, Android tracker](../mobile-trackers/hybrid-apps/index.md), or [React Native](../react-native-tracker/hybrid-apps/index.md)).
+The diagram below shows the interaction of the WebView and mobile trackers in hybrid apps.
+
+```mermaid
+flowchart TB
+
+subgraph hybridApp[Hybrid Mobile App]
+
+    subgraph webView[Web View]
+        webViewCode[App logic]
+        webViewTracker[Snowplow WebView tracker]
+
+        webViewCode -- "Tracks events" --> webViewTracker
+    end
+
+    subgraph nativeCode[Native Code]
+        nativeAppCode[App logic]
+        nativeTracker[Snowplow iOS/Android/React Native tracker]
+
+        nativeAppCode -- "Tracks events" --> nativeTracker
+    end
+
+    webViewTracker -- "Forwards events" --> nativeTracker
+end
+
+subgraph cloud[Cloud]
+    collector[Snowplow Collector]
+end
+
+nativeTracker -- "Sends tracked events" --> collector
+```
```

Comment:
This doesn't seem to be working on the preview: https://deploy-preview-102--snowplow-docs.netlify.app/docs/collecting-data/collecting-from-own-applications/webview-tracker/

Reply:
Not sure why that is, maybe the preview is not updating installed packages? cc @stanch

Reply:
I’ve rebuild the preview without cache — same result. Then I tried it locally — same result.
My conclusion is that you forgot `remarkPlugins: [mdxMermaid.default]` :)

Reply:
Works now